homeassistant/components/light/hue.py

```diff
@@ -340,8 +361,41 @@ def turn_off(self, **kwargs):
         elif self.bridge_type == 'hue':
             command['alert'] = 'none'
 
-        self.bridge.set_light(self.light_id, command)
+        self._command_func(self.light_id, command)
 
     def update(self):
         """Synchronize state with bridge."""
         self.update_lights(no_throttle=True)
+
+
+class HueLightGroup(HueLight):
+    """Representation of a Hue light group."""
+
+    def __init__(self, light_id, info, bridge, update_lights,
+                 bridge_type):
+        """Initialize the light group."""
+        super().__init__(light_id, info, bridge, update_lights,
+                         bridge_type, False)
+        self._command_func = self.bridge.set_group
+
+    @property
+    def brightness(self):
+        """Return the brightness of this light group between 0..255."""
+        return self.info['action'].get('bri')
```

Comment:
It's the same attribute name, just a different position in a dict. So you can just store the correct dict in the update method and you won't have to override the properties.